### ISSUE TITLE
unable to default to buyer renderer

### ISSUE THREAD

**Issue by patmmccann** [prebid/Prebid.js#2668]
## Type of issue
Feature request

## Description
We would like to prefer the buyer's outstream renderer when it is available and only provide our own as a backup. Rendering is expensive and if the buyer is happy to pay for it, we are happy to let them. The wiki says we are leaving money on the table by not providing a renderer though. 

In the wiki: 
Prebid.js will select the renderer used to display the outstream video in the following way:

If a renderer is associated with the Prebid adUnit, it will be used to display any outstream demand associated with that adUnit. Below, we will provide an example of an adUnit with an associated renderer.
If no renderer is specified on the Prebid adUnit, Prebid will invoke the renderer associated with the winning (or selected) demand partner video bid.
At this time, since not all demand partners return a renderer with their video bid responses, we recommend that publishers associate a renderer with their Prebid video adUnits, if possible. By doing so, any Prebid adapter that supports video will be able to provide demand for a given outstream slot.

**Comment by YOzaz** [comment]
+1 to this one.

**Comment by patmmccann** [comment]
Also, this behavior would enable deals with buyers that want to bring their own player as well as taking advantage of demand that doesnt come with a player. I imagine this would be the default instead of unsupported.

**Comment by YOzaz** [comment]
Guys, in fact - this is very actual when multiple SSPs are used. A good example would be Prebid.js config with such adapters enabled:
* Rubicon
* Unruly

Unruly have their own renderer, while Rubicon doesn't. Unruly prefers to use their own renderer and are not charging for it. Rubicon suggests using 3rd party player with active transcoding in place (which is why video.js is not suitable), and such players are usually paid - e.g. BBW.

In this case, it would be ideal to use Unruly renderer when Unruly bid comes through, and switch to Rubicon + BBW (but adjusting bid CMP to take away BBW costs) otherwise.
With current Prebid.js configuration, it is, unfortunately, impossible.

**Comment by YOzaz** [comment]
Hi guys, any updates on this? Rubicon does not have, and is not planning to add their player to outstream video bids. This basically leaves a hard choice on bidding - either Unruly + SpotX + Teads (e.g.), either Rubicon alone.

**Comment by nanek** [comment]
As a work-around, I was able to add the adUnit twice, once with a renderer and another time without a renderer. I'm not sure if this creates unintended side effects, however it seems allows you to run a custom renderer for one partner, and the provided partner renderer for another partner.

Something like:

````
pbjs.addAdUnit([{
    code: slot.code,
    bids: [], // Add Rubicon
    mediaTypes: {
        video: {
            context: 'outstream',
            playerSize: [640, 480]
        }
    },
    renderer: {} // with renderer
},
{
    code: slot.code,
    bids: [], // Add teads without renderer
    mediaTypes: {
        video: {
            context: 'outstream',
            playerSize: [640, 480]
        }
    },
}])
````

However, I think this feature request is still a good one. In particular, this would not cover the case if some bids from a single partner return a renderer and others do not.

**Comment by YOzaz** [comment]
@nanek what if both partners respond with a bid? Are they being put into CPM auction?

**Comment by patmmccann** [comment]
solved bt #5638
